src/main/java/com/questhelper/quests/beneathcursedsands/BeneathCursedSands.java

```diff
@@ -0,0 +1,591 @@
+		finishQuest = new NpcStep(this, NpcID.MAISA_11474, new WorldPoint(3281, 2772, 0), "Talk to Maisa or the High Priest in Sophanem to finish the quest.");
+		((NpcStep) finishQuest).addAlternateNpcs(NpcID.HIGH_PRIEST_4206);
+	}
+
+	@Override
+	public List<ItemRequirement> getItemRecommended()
+	{
+		return Arrays.asList(waterskins, antipoison, accessToFairyRings, pharaosSceptre, combatGear, food, staminaPotions, nardahTeleport);
+	}
+
+	@Override
+	public List<ItemRequirement> getItemRequirements()
+	{
+		return Arrays.asList(coal, tinderbox, ironBar, spade, meat);
+	}
+
+	@Override
+	public List<String> getCombatRequirements()
+	{
+		return Arrays.asList("Head Menaphite Guard (174) without protection prayers", "Two Scabarite Mages (119)", "Champion of Scabaras (379)", "Menaphite Akh (351)");
+	}
+
+	@Override
+	public QuestPointReward getQuestPointReward()
+	{
+		return new QuestPointReward(2);
+	}
+
+	@Override
+	public List<ExperienceReward> getExperienceRewards()
+	{
+		return Collections.singletonList(new ExperienceReward(Skill.AGILITY, 20000));
+	}
+
+	@Override
+	public List<UnlockReward> getUnlockRewards()
+	{
+		return Arrays.asList(new UnlockReward("Access to the Tombs of Amascut."), new UnlockReward("Ability to unlock the Jaltevas teleport option on the Pharaoh's sceptre"), new UnlockReward("Access to fairy ring code A.K.P (a small island south-west of Necropolis)"));
+	}
+
+	@Override
+	public List<ItemReward> getItemRewards()
+	{
+		return Arrays.asList(new ItemReward("A Keris Partisan", ItemID.KERIS_PARTISAN, 1), new ItemReward("A Circlet of Water", ItemID.CIRCLET_OF_WATER, 1));
+	}
+
+	@Override
+	public List<PanelDetails> getPanels()
+	{
+		List<PanelDetails> allSteps = new ArrayList<>();
+		allSteps.add(new PanelDetails("Starting off", Arrays.asList(talkToJamilaToStart, readMessage), Collections.emptyList(), Collections.singletonList(waterskins)));
+		allSteps.add(new PanelDetails("Entranced Menaphites", Arrays.asList(talkToMaisaStartInvestigation, inspectBlockedPyramidEntry, talkToCitizenOrGuard, fightHeadMenaphiteGuard), Collections.singletonList(combatGear), Arrays.asList(waterskins, food)));
+		allSteps.add(new PanelDetails("The Ruins of Ullek", Arrays.asList(talkToMaisaExploreCliffs, inspectFurnace, useCoalOnFurnace, useTinderboxOnFurnace, searchWell, readStoneTablet, digForChest, openChest, craftEmblem, useEmblemOnPillar, confirmScarabRotation, enterDungeonToFightScarabMages, fightScarabMages, climbDownStairsAgain, pullLever, pullSecondLever, enterRiddleDoor), Arrays.asList(combatGear, coal, tinderbox, spade, ironBar), Arrays.asList(food, antipoison, waterskins)));
+		allSteps.add(new PanelDetails("Riddle of the Tomb", Arrays.asList(obtainEmblemsFromPlaque, placeEmblemsInUrns, pullRiddleLever, enterTombDoor, talkToSpirit, takeRustyKey)));
+		allSteps.add(new PanelDetails("The Champion of Scabaras", Arrays.asList(unlockBossDoor, fightChampionOfScabaras, talkToScabarasHighPriest), Arrays.asList(combatGear, food, rustyKey)));
```

Comment:
`combatGearChampionOfScabaras`? Players should know which combat gear to bring before leaving for the temple to avoid having to leave and return

Reply:
Good catch. Added.